clappr/clappr-core
# Fix leaks

## Summary

This PR fixes a leak of the `contextmenu` event handler on the video tag and prevents adding multiple `style` tag children to the `core` when changing the video with the `load` method.

## Changes

- `core.js`:
  - Add style only once
- `html5_video.js`:
  - Remove `contextmenu` on destroy

## How to test

1. Play the sample media
2. Run `load` on the same sample media many times
3. There should be no duplicated `style` tags and only one `contextmenu` listener

## Images

### Before this PR

![Screenshot 2023-08-28 at 14 01 46](https://github.com/clappr/clappr-core/assets/14154066/d8237bfb-dcc8-4bcf-a830-0bba9d5f8526)


### After this PR

![Screenshot 2023-08-28 at 14 02 31](https://github.com/clappr/clappr-core/assets/14154066/b9ef4741-8cc7-4670-8e64-1263e04cf2e0)
